kythe/go/extractors/config/wrapper/gradle_build_modifier_test.go

```diff
@@ -98,16 +98,16 @@ func TestPreprocess(t *testing.T) {
 		}
 
 		// Compare results.
-		eq, diff := testutil.TrimmedEqual(readBytes(t, tfName), readBytes(t, getPath(tcase.expectedOutputFile)))
+		eq, diff := testutil.TrimmedEqual(readBytes(t, tfName), readBytes(t, getPath(t, tcase.expectedOutputFile)))
 		if !eq {
 			t.Errorf("Expected input file %s to be %s, but got diff %s", tcase.inputFile, tcase.expectedOutputFile, diff)
 		}
 	}
 }
 
 // getPath returns a resolved filepath name for a file living in testdata/ directory.
-func getPath(f string) string {
-	return os.ExpandEnv(filepath.Join(testDataDir, f))
+func getPath(t *testing.T, f string) string {
+	return testutil.TestFilePath(t, filepath.Join(testDataDir, f))
```

Comment:
If you're going to pass in the `*testing.T`, please also add `t.Helper()` at the top.
(It ensures logs from the function get attributed to the right location)

Reply:
done